config.yaml

```diff
@@ -562,6 +562,24 @@ usage_collector:
     # Sets the interval between running the usage collector in days
     interval_in_days: 1
 
+prometheus:
+  port: 9090
+  node_exporter:
+    metrics_port: 9100
+  blackbox_exporter:
+    metrics_port: 9115
+  postgres_exporter:
+    username: postgres
+    password: postgres
+    sslmode: disable
+    metrics_port: 9187
+  rabbitmq_exporter:
+    rabbitmq_url: https://127.0.0.1:15671
+    username: cloudify
```

Comment:
i think keep this empty, and on the python side, default it to `config['rabbitmq']['username']` etc
maybe we don't even need a separate setting for this, and it can just always be rabbitmq.username instead?

Reply:
ditto postgres